### PR TITLE
Clarify sentence meaning

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Clarify meaning in docs that dbt run / dbt build is required to view models in Explorer

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Needs review from product team

### CODE REVIEW

File: website/docs/docs/collaborate/explore-projects.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ Try dbt Explorer! It's available in [Public Preview](/docs/dbt-versions/product-
...
-- Run [dbt test](/reference/commands/test) or [dbt build](/reference/commands/build) within a job in the environment to view test results.
-- Run [dbt source freshness](/reference/commands/source#dbt-source-freshness) within a job in the environment to view source freshness data.
-- Run [dbt snapshot](/reference/commands/snapshot) or [dbt build](/reference/commands/build) within a job in the environment to view snapshot details.
+To view a resource and its metadata, you must define the resource in your project and run a job in the production environment. The resulting metadata depends on the [commands](/docs/deploy/job-commands) executed by the jobs.
+
+| To view in Explorer | You must successfully run |
+|---------------------|---------------------------|
+| Model details or results | [dbt run](/reference/commands/run) or [dbt build](/reference/commands/build) on a given model within a job in the environment |
+| Catalog statistics and columns for models, sources, and snapshots | [dbt docs generate](/reference/commands/cmd-docs) within a job in the environment |
```
Comment (cafzal) [inline website/docs/docs/collaborate/explore-projects.md]: Please reword: "Columns and statistics for models, sources, and snapshots"

File: website/docs/docs/collaborate/explore-projects.md [hunk 2]
```diff
@@ -26,15 +26,15 @@ Try dbt Explorer! It's available in [Public Preview](/docs/dbt-versions/product-
...
+| Test results | [dbt test](/reference/commands/test) or [dbt build](/reference/commands/build) within a job in the environment | 
+| Source freshness data | [dbt source freshness](/reference/commands/source#dbt-source-freshness) within a job in the environment |
+| Snapshot details | [dbt snapshot](/reference/commands/snapshot) or [dbt build](/reference/commands/build) within a job in the environment |
```
Comment (dave-connors-3) [inline website/docs/docs/collaborate/explore-projects.md:37]: I think we need a corresponding row for seeds to, with the commands `dbt seed` or `dbt build`

File: website/docs/docs/collaborate/explore-projects.md [hunk 3]
```diff
@@ -26,15 +26,15 @@ Try dbt Explorer! It's available in [Public Preview](/docs/dbt-versions/product-
...
-- Run [dbt docs generate](/reference/commands/cmd-docs) within a job in the environment to view catalog statistics and columns for models, sources, and snapshots.
-- Run [dbt test](/reference/commands/test) or [dbt build](/reference/commands/build) within a job in the environment to view test results.
-- Run [dbt source freshness](/reference/commands/source#dbt-source-freshness) within a job in the environment to view source freshness data.
-- Run [dbt snapshot](/reference/commands/snapshot) or [dbt build](/reference/commands/build) within a job in the environment to view snapshot details.
+To view a resource and its metadata, you must define the resource in your project and run a job in the production environment. The resulting metadata depends on the [commands](/docs/deploy/job-commands) executed by the jobs.
+
+| To view in Explorer | You must successfully run |
+|---------------------|---------------------------|
+| Model details or results | [dbt run](/reference/commands/run) or [dbt build](/reference/commands/build) on a given model within a job in the environment |
```
Comment (cafzal) [inline website/docs/docs/collaborate/explore-projects.md]: @nghi-ly Please change to "Model lineage, details, or results" per @dave-connors-3 suggestion

File: website/docs/docs/collaborate/explore-projects.md (hunk 2 above)
Comment (cafzal) [reply website/docs/docs/collaborate/explore-projects.md:37]: Makes sense! Worth to add a row for seeds then @nghi-ly

File: website/docs/docs/collaborate/explore-projects.md [hunk 4]
```diff
@@ -26,15 +26,15 @@ Try dbt Explorer! It's available in [Public Preview](/docs/dbt-versions/product-
...
+| Columns and statistics for models, sources, and snapshots| [dbt docs generate](/reference/commands/cmd-docs) within a job in the environment |
+| Test results | [dbt test](/reference/commands/test) or [dbt build](/reference/commands/build) within a job in the environment | 
+| Source freshness data | [dbt source freshness](/reference/commands/source#dbt-source-freshness) within a job in the environment |
```
Comment (cafzal) [inline website/docs/docs/collaborate/explore-projects.md]: Please change to "Source freshness results" @nghi-ly for clarity

File: website/docs/docs/collaborate/explore-projects.md [hunk 5]
```diff
@@ -26,15 +26,16 @@ Try dbt Explorer! It's available in [Public Preview](/docs/dbt-versions/product-
...
+To view a resource and its metadata, you must define the resource in your project and run a job in the production environment. The resulting metadata depends on the [commands](/docs/deploy/job-commands) executed by the jobs.
+
+| To view in Explorer | You must successfully run |
+|---------------------|---------------------------|
+| Model lineage, details, or results | [dbt run](/reference/commands/run) or [dbt build](/reference/commands/build) on a given model within a job in the environment |
+| Columns and statistics for models, sources, and snapshots| [dbt docs generate](/reference/commands/cmd-docs) within a job in the environment |
+| Test results | [dbt test](/reference/commands/test) or [dbt build](/reference/commands/build) within a job in the environment | 
+| Source freshness results | [dbt source freshness](/reference/commands/source#dbt-source-freshness) within a job in the environment |
+| Snapshot details | [dbt snapshot](/reference/commands/snapshot) or [dbt build](/reference/commands/build) within a job in the environment |
+| Seeds | [dbt seed](/reference/commands/seed) or [dbt build](/reference/commands/build) within a job in the environment |
```
Comment (dave-connors-3) [inline website/docs/docs/collaborate/explore-projects.md]:
```suggestion
| Seed details | [dbt seed](/reference/commands/seed) or [dbt build](/reference/commands/build) within a job in the environment |
```

**Review by dave-connors-3 — APPROVED** [review]
🦞

**Pull request merged**